dockers/docker-fpm-frr/Dockerfile.j2

```diff
@@ -18,7 +18,8 @@ RUN apt-get update   && \
         iproute2        \
         libjson-c3      \
         logrotate       \
-        libunwind8
+        libunwind8      \
+	net-tools
```

Comment:
Is `net-tools` still necessary here?

Reply:
I have one doubt. I am deleting the /etc/sonic/config_db.json and rebooting the system. After reboot, I expect that config db and vtysh config should be empty. But I see that we still have BGP entries in redis DB. e.g.", 
router bgp 65100
 neighbor 10.0.0.1 remote-as 65200"
Can you suggest where from these entries are coming after reboot. It is expected that when we delete config_db.json and rebooting, the redis DB should also become empty.

Reply:
Could you check if /etc/sonic/minigraph.xml exists? I think if config_db.json does not exist, it will try to recover the configuration from minigraph.

Reply:
I have checked this file. It is not present in /etc/sonic, but at some other place. But even after deleting the file and deleting /etc/sonic/config_db.json and reboot, I still see that all BGP configurations are retained. I don't know from where this config is getting replayed. I am using default mode (separated).

Reply:
Is there anything (e.g., config_db.json, minigraph.xml) in /host/old_config?